sdcm/remote/kubernetes_cmd_runner.py

```diff
@@ -294,10 +294,20 @@ def _read_from_stream(self, num_bytes: int) -> str:
         pod_name = self.context.config.k8s_pod_name
         with self._ws_lock:
             if self.process.closed:
-                LOGGER.debug(
-                    "'_read_from_stream' called for the %s pod having closed socket. Recreating it.",
-                    pod_name)
-                self._start()
+                self.pod_counter_to_live -= 1
+                if self.pod_counter_to_live < 1:
+                    LOGGER.warning(
+                        "'_read_from_stream': stopping '%s' pod because stream to it "
+                        "cannot be established having alive pod.",
+                        self.context.config.k8s_pod_name)
+                    self._stop_pod()
+                    time.sleep(10)
```

Comment:
Why the `10 seconds sleep` is needed here?

Reply:
before pod is stopped, reader is still operating causing some unnecessary log messages.
I was thinking that anyway if we kill the pod due some freeze/too many reconnections, we don't care about another 10 seconds.